vllm/entrypoints/openai/protocol.py

```diff
@@ -57,12 +57,14 @@ class ChatCompletionRequest(BaseModel):
     messages: Union[str, List[Dict[str, str]]]
     temperature: Optional[float] = 0.7
     top_p: Optional[float] = 1.0
+    min_p: Optional[float] = 0.0
```

Comment:
Can you move the parameters to the section below under
```python
    # Additional parameters supported by vLLM
```

Reply:
Didn't notice that..:) Resolved.